canonical-web-and-design/ubuntu.com
# Grid updates pt2

## Done

- updated grid layouts on /desktop, /download, /embedded, /esm and /financial-services pages in line with Vanilla 2's new grid patterns (see point 2 of [the upgrade guide](https://ubuntu.com/blog/vanilla-framework-2-0-upgrade-guide))

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the grid/column layouts on these pages look correct/not broken:
  - /desktop
  - /desktop/contact-us
  - /desktop/developers
  - /desktop/features
  - /desktop/organisations
  - /desktop/partners
  - /desktop/statistics
  - /desktop/thank-you
  - /download
  - /download/cloud
  - /download/desktop
  - /download/desktop/thank-you
  - /download/iot
  - /download/iot/installation
  - /download/iot/intel-iei-tank-870
  - /download/iot/intel-joule-desktop
  - /download/iot/intel-joule
  - /download/iot/intel-nuc-desktop
  - /download/iot/intel-nuc
  - /download/iot/kvm
  - /download/iot/orange-pi-zero
  - /download/iot/qualcomm-dragonboard-410c
  - /download/iot/raspberry-pi-2-3-core
  - /download/iot/raspberry-pi-2-3
  - /download/iot/raspberry-pi-compute-module-3
  - /download/iot/samsung-artik-5-10-server
  - /download/iot/samsung-artik-5-10
  - /download/iot/up-squared-iot-grove-server
  - /embedded
  - /esm
  - /financial-services

## Issue / Card

Fixes #5668
